### DISCUSSION TITLE
reverse title and arrow icon

### DISCUSSION THREAD

**Discussion by abdelhak-ihadjadene** [saadeghi/daisyui#2938] [Q&A] [answered]
I wonder how someone can reverse the layout of the title and the icon, for an Arabic layout, for example, we would have the icon on the left side and the title on the right.

**Comment by saadeghi** [comment, marked as answer]
You can simply put an icon and a text next to each other and they will be RTL on an RTL page.  
You can also put an icon and a text in a div with `flex` class.

Let me know if you have any questions
